controls/templatedpicker/commands.md

```diff
@@ -18,22 +18,23 @@ The TemplatedPicker for .NET MAUI exposes the following commands, which enable y
 * `ToggleCommand`&mdash;Allows you to show/hide the popup. Used for selecting an item from the custom picker.
 * `ClearCommand`&mdash;Allows you to clear the displayed item.
 
-## PopupSelector Commands
+## Settings Commands
```

Comment:
Please see the DatePicker or the DateTimePicker for the suggested changes in this section.